src/scheduler/data_types.h

```diff
@@ -1069,7 +1069,7 @@ struct config
 	fairshare_head *fairshare;		/* fairshare tree */
 	time_t decay_time;			/*  time in seconds for the decay period*/
 	time_t sync_time;			/* time between syncing usage to disk */
-	struct t prime[HIGH_DAY][HIGH_PRIME];	/* prime time start and prime time end*/
+	struct t prime[HIGH_DAY][2];	/* prime time start and prime time end*/
```

Comment:
Why the change from HIGH_PRIME to 2?

Reply:
the other 2 rows (corresponding to ALL and NONE) were not used, so it was just unused memory.

Reply:
This will probably never happen, but the idea was if we ever add more time windows than just prime and non-prime, this will just automatically update itself.  We'll add a new value to the enum and HIGH_PRIME will increase.  The amount of lost memory is pretty minor.